beard.js

```diff
@@ -3,6 +3,17 @@ const exts = '(.beard$)';
 const traversy = require('traversy');
 const normalize = require('path').normalize;
 
+class BeardError {
+  constructor(realError, template, lineNumber, tag) {
+    this.name = 'BeardError';
+    Error.captureStackTrace(this, this.constructor);
```

Comment:
Can you put this at the end of this constructor? Also why are you passing in this.constructor into the captureStatckTrace?

Reply:
@shanebo https://nodejs.org/api/errors.html#errors_error_capturestacktrace_targetobject_constructoropt. Basically, that argument means that the `constructor` function will not show up in the stack trace.